lib/external_upload_helpers.rb

```diff
@@ -136,7 +136,8 @@ def batch_presign_multipart_parts
       ExternalUploadStub.find_by(unique_identifier: unique_identifier, created_by: current_user)
     return render_404 if external_upload_stub.blank?
 
-    return render_404 if !multipart_upload_exists?(external_upload_stub)
+    exists, message = multipart_upload_exists?(external_upload_stub)
```

Comment:
Since this only has two return signatures: `[false, message]` or `[true, nil]`, we could simplify it to just return the message (or not), and scrap the suffix `?` to avoid confusion.

Reply:
Yeah it's kind of an ugly duckling this one...what would you suggest as a different name? `multipart_upload_exists` seems a bit odd. Maybe `check_multipart_upload_exists`?

Reply:
I'll do `check_multipart_upload_exists`